mmseg/models/decode_heads/segmenter_mask_head.py

```diff
@@ -0,0 +1,98 @@
+import torch
+import torch.nn as nn
+from einops import rearrange
+from mmcv.cnn import build_norm_layer
+from mmcv.cnn.utils.weight_init import trunc_normal_
+from mmcv.runner import ModuleList
+
+from mmseg.models.backbones.vit import TransformerEncoderLayer
+from ..builder import HEADS
+from .decode_head import BaseDecodeHead
+
+
+@HEADS.register_module()
+class SegmenterMaskTransformerHead(BaseDecodeHead):
+
```

Comment:
Add docstring to summarize `SegmenterMaskTransformerHead` behavior and document its arguments.  like
https://github.com/open-mmlab/mmsegmentation/blob/bdfbaf00c1f55e054e6088e60f955345ab7383c0/mmseg/models/decode_heads/fpn_head.py#L12-L24